apps/backend/tests/integration/test_data_isolation.py

```
@@ -0,0 +1,103 @@
+    """All data endpoints must return 401 without a Bearer token."""
+
+    PROTECTED_ENDPOINTS = [
+        ("GET", "/api/v1/resumes?resume_id=fake"),
+        ("GET", "/api/v1/resumes/list"),
+        ("GET", "/api/v1/resumes/fake/pdf"),
+        ("DELETE", "/api/v1/resumes/fake"),
+        ("GET", "/api/v1/resumes/fake/job-description"),
+        ("GET", "/api/v1/jobs/fake"),
+        ("GET", "/api/v1/status"),
+        ("POST", "/api/v1/enrichment/analyze/fake"),
+    ]
```

Ruff (0.15.9)

[warning] Mutable default value for class attribute

(RUF012)